unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Audio/AudioContainer.cs

```diff
@@ -29,9 +30,32 @@ void Awake()
             maxDistance = 400f;
         }
 
+        if (instantiateEvents)
+        {
+            for(int i = 0; i < audioEvents.Count; i++)
+            {
+                string str = audioEvents[i].name;
+                AudioEvent instance = Instantiate(audioEvents[i]);
+                audioEvents[i] = instance;
+                instance.name = str;
+            }
+        }
+
         foreach (AudioEvent e in audioEvents)
         {
             e.Initialize(this);
         }
     }
+
+    public AudioEvent GetEvent(string eventName)
+    {
+        for (int i = 0; i < audioEvents.Count; i++)
+        {
+            if (audioEvents[i].name == eventName)
+                return audioEvents[i];
+        }
+
+        Debug.Log(name + "'s AudioContainer couldn't find an event called " + eventName);
```

Comment:
```suggestion
        Debug.Log( $"{name}'s AudioContainer couldn't find an event called {eventName}");
```
We can use the more readable interpolated strings here